app/src/hooks/useTransfersEstimate.tsx

```diff
@@ -0,0 +1,52 @@
+import { Direction } from '@/services/transfer'
+import useSnowbridgeContext from './useSnowbridgeContext'
+import { useQuery } from '@tanstack/react-query'
+import { getSnowBridgeEtimatedTransferDuration } from '@/context/snowbridge'
+import { Context } from '@snowbridge/api'
+
+export type EstimatedTransferDuration = {
```

Comment:
I suggested above considering just making this a const.

```suggestion
export const ESTIMATED_DURATION = {
```